operations/CSM_product_management/Redeploying_a_Chart.md

```diff
@@ -0,0 +1,214 @@
+# Redeploying a Chart
```

Comment:
This entire document feels like it should be boiled down to 1 or 2 scripts, whose use can be referenced somewhere else.

Reply:
I agree. The problem is:
1) Scripts require far more testing than manual instructions, since you have to check for things that a human would obviously realize are problems but that a script would not.
2) It will take me longer just to write the scripts.
3) We want to avoid adding scripts to docs-csm and instead put them in libcsm, but for that repo we are trying not to add any scripts which don't have accompanying unit tests. That makes it more work again.

I think automation is the eventual solution.

I cannot repeat enough that the goal here is to provide a bandaid that fixes the problems we have seen with the existing procedures -- namely, that they overwrite the manifests in Kubernetes, and that they do not track cases where version numbers have been updated due to hotfixes. It is a secondary step to provide a more elegant and proper solution to this. I don't have the time to dedicate to that right now. That said, if you do happen to have time to tackle it, I am happy to assign this over to you. :)